### PR TITLE
Everything looks better with name in it

### PR DESCRIPTION
**Pull request by PeterBell**

### CODE REVIEW

**Comment by PeterBell** [comment]
@brntbeer @reimundo @tlberglund - would love to get your input on whether this is the best way to add my name to the site :smile:

**Comment by reimundo** [comment]
Search and replace

**Comment by brntbeer** [comment]
Step 1: add actual content

**Comment by PeterBell** [comment]
I don't think it's there, so replacing would't work very wekk

**Comment by bruceflowers** [comment]
I think your name needs to be in there several more times.

**Comment by PeterBell** [comment]
Wait, I need to do _everything_ on this team? :smile:

**Comment by PeterBell** [comment]
@bruceflowers yay - :+1:

**Comment by PeterBell** [comment]
> I think your name needs to be in there several more times.

Agreed :+1:

**Comment by bruceflowers** [comment]
> Search and replace

replace with search

**Comment by reimundo** [comment]
> Step 1: add actual content

@brntbeer - Good point

File: peterbell.txt [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Peter Bell
+Peter Bell
+Peter Bell
+Peter Bell
+Peter Bell
+Peter Bell
```
Comment (brntbeer) [inline peterbell.txt:6]: Ego trip.

File: peterbell.txt [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
```
Comment (PeterBell) [inline peterbell.txt:4]: Is this copy of my name the same as the others?

File: peterbell.txt [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+Peter Bell
```
Comment (reimundo) [inline peterbell.txt:5]: This one seems different

File: peterbell.txt (hunk 1 above)
Comment (brntbeer) [inline peterbell.txt:6]: Add more comments

File: peterbell.txt [hunk 4]
```diff
@@ -0,0 +1,7 @@
...
```
Comment (bruceflowers) [inline peterbell.txt:3]: Third time's the charm.

**Comment by brntbeer** [comment]
:+1: :ship:

**Comment by brntbeer** [comment]
#4 is similar work to what we want here

**Pull request merged**